conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -11,7 +11,7 @@
 
 # Add all the application routes to the prod.routes file
 ->         /                                              prod.Routes
-GET        /test-only/reset-registrations                 uk.gov.hmrc.softdrinksindustrylevy.controllers.test.TestingController.resetRegistrations
-GET        /test-only/reset-returns                       uk.gov.hmrc.softdrinksindustrylevy.controllers.test.TestingController.resetReturns
+GET        /test-only/reset-registrations                 uk.gov.hmrc.softdrinksindustrylevy.controllers.test.TestingController.reset(url = "reset-registration")
+GET        /test-only/reset-returns                       uk.gov.hmrc.softdrinksindustrylevy.controllers.test.TestingController.reset(url = "reset-returns")
```

Comment:
why not `GET /test-only/:url uk.gov.hmrc.whyisthissolong.controllers.test.TestingController.reset(url)`?

Reply:
Yes very tasty